GitHubContents.ps1

```diff
@@ -322,7 +325,6 @@ filter Set-GitHubContent
         [string] $CommitMessage,
 
         [Parameter(
-            Mandatory,
```

Comment:
This needs to remain mandatory in its own `ParameterSet`.

Reply:
Fixed

Reply:
Take a look at `Get-GitHubEvent`.  That's an example where we had to explode the number of ParameterSets in order to support both the `Elements` and `Uri` repo scenarios while still accurately separating out the different possible inputs for the method itself.